diff --git a/index.js b/index.js
index d695233eda4652b4c554a0967970295d7b9cfdaa..e4121c7e5f9afc7fb01337b84c24fc754e415aa7 100644
--- a/index.js
+++ b/index.js
@@ -9,10 +9,7 @@ module.exports = inquire;
  */
 function inquire(moduleName) {
   try {
-    if (typeof require !== "function") {
-      return null;
-    }
-    var mod = require(moduleName);
+    var mod = eval("require")(moduleName); // eslint-disable-line no-eval
     if (mod && (mod.length || Object.keys(mod).length)) return mod;
     return null;
   } catch (err) {
@@ -20,18 +17,3 @@ function inquire(moduleName) {
     return null;
   }
 }
-
-/*
-// maybe worth a shot to prevent renaming issues:
-// see: https://github.com/webpack/webpack/blob/master/lib/dependencies/CommonJsRequireDependencyParserPlugin.js
-// triggers on:
-// - expression require.cache
-// - expression require (???)
-// - call require
-// - call require:commonjs:item
-// - call require:commonjs:context
-
-Object.defineProperty(Function.prototype, "__self", { get: function() { return this; } });
-var r = require.__self;
-delete Function.prototype.__self;
-*/
